CHANGES

```diff
@@ -1,11 +1,35 @@
 CHANGES file for iptraf-ng
 
-* Philip Cameron <pcameron@redhat.com> - 1.1.5
-- Incorporate numerous cleanup changes in the code
-- Revised documentation. Previous distributions shipped the
-  original IPTraf documentation.
-- Bugfix: segfault in rate_add_rate
-- Bugfix: Fix memory leak when destroying iflist
+* Mon Jun 01 2020 Vitezslav Samel <vitezslav@samel.cz> - 1.2.0
+- ipfrag: code refactoring (Nikola Pajkovsky)
+- ifstats: sort interfaces by name (Jan Engelhardt)
+- ifstats: fix memory leak (Phil Cameron)
```

Comment:
@vsamel 
I fixed issue 23 and it needs to be in release 1.2
ipfilter - fix display of internal memory (issue 23) (Phil Cameron)
Please add above or similar.